<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentificationEditor</class>
 <widget class="QWidget" name="IdentificationEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelmage">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonValider">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../ressources.qrc">
         <normaloff>:/icones/validate.png</normaloff>:/icones/validate.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonInvalider">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../ressources.qrc">
         <normaloff>:/icones/invalidate.png</normaloff>:/icones/invalidate.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPersonne">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonIdentifier">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../ressources.qrc">
         <normaloff>:/icones/identify.png</normaloff>:/icones/identify.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonIgnorer">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../ressources.qrc">
         <normaloff>:/icones/ignore.png</normaloff>:/icones/ignore.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
